101-hdinsight-secure-vnet/azuredeploy.json

```diff
@@ -62,27 +64,27 @@
       "type": "Microsoft.Network/networkSecurityGroups",
       "name": "[variables('networkSecurityGroup').name]",
       "location": "[parameters('location')]",
-      "apiVersion": "2016-03-30",
+      "apiVersion": "2017-06-01",
       "properties": {
         "securityRules": [
           {
-            "name": "hdirule1",
+            "name": "Allow_HDInsight_Management_Traffic",
             "properties": {
-              "description": null,
+              "description": "Allow traffic from the Azure health and management services to reach HDInsight",
               "protocol": "*",
               "sourcePortRange": "*",
               "destinationPortRange": "443",
-              "sourceAddressPrefix": "168.61.49.99",
+              "sourceAddressPrefixes": ["168.61.49.99", "23.99.5.239", "168.61.48.131", "138.91.141.162"],
```

Comment:
what's the source of the ip addresses here?

Reply:
The IPs in question are listed at https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-extend-hadoop-virtual-network. I tried adding that link to the description, but it is limited to 140 chars :(